frontend/public/components/dashboards-page/overview-dashboard/health-card.tsx

```diff
@@ -149,6 +165,21 @@ const HealthCard_: React.FC<HealthProps> = ({
           />
         </HealthBody>
       </DashboardCardBody>
+
+      {alerts.length > 0 &&
+        <React.Fragment>
+          <DashboardCardHeader className="co-health-card__alerts-border">
+            <DashboardCardTitle>Alerts</DashboardCardTitle>
+          </DashboardCardHeader>
+          <DashboardCardBody isLoading={flagPending(openShiftFlag)}>
+            <AlertsBody>
+              {alerts.map(alert => (
+                <AlertItem key={alert.fingerprint} alert={alert} />
```

Comment:
I think it should be `alert.annotations.fingerprint`. 

@rawagner thoughts?

Reply:
Its correct AFAIK. `alert.fingerprint`

Reply:
@rawagner I verified its `alert.fingerprint`. Thanks